Todo-css.css

```diff
@@ -122,9 +122,9 @@ input {
 
 /* Stil za tekst unutar popup prozora */
 .error-popup {
-  font-size: 24px;
-  margin-bottom: 10px;
-  background-color: #ff0000;
-  color: #fff;
-  padding: 100px;
+  font-size: 24px; /* Promenite željenu veličinu fonta */
+  margin-bottom: 10px; /* Dodajte razmak ispod teksta */
+  background-color: #ff0000; /* Promenite boju pozadine prema želji */
+  color: #fff; /* Promenite boju teksta prema želji */
+  padding: 100px; /* Dodajte padding za unutar popup prozora */
```

Comment:
You shouldnt comment like this, because who ever knows CSS knows what each line do, so it is redundant. Only can do it for something that you think might confuse a developer.